package.xml

```diff
@@ -409,8 +437,7 @@ https://github.com/php/pecl-search_engine-solr
   <dependencies>
     <required>
       <php>
-        <min>7.0</min>
-        <max>8.2.99</max>
+        <min>7.4</min>
```

Comment:
I was thinking you want to keep 7.0+
In this case (7.4+) we can switch to PKG_CHECK_MODULES which simplify a bit the config.m4

BTW, no urgency, this can be done later

Reply:
BTW: on Windows 7.0+ is still supported.

Reply:
@Jan-E config.32 is different, so all good

Reply:
After some discussion we think that we will bump the minimum version to 7.4 in this new release.
We will fix the config.m4 file later since we want to finish this release today.